dynamic-demo-plugin/src/components/Modals/ModalPage.tsx

```diff
@@ -89,7 +89,7 @@ const LoadingComponent: React.FC = () => {
 type TestOverlayComponentProps = {
   heading?: string;
 };
-
+// @ts-ignore
```

Comment:
if we really need TS to ignore this, prefer `ts-expect-error` as it will alert you when the statement is no longer needed

```suggestion
// @ts-expect-error - [insert reason for ts-expect-error here]
```